src/nasher/compile.nim

```diff
@@ -161,7 +180,10 @@ proc compile*(opts: Options, pkg: PackageRef): bool =
 
       for file in walkFiles("*.nss"):
         files.add(file)
-        if file notin pkg.updated and file.executable:
+        if file.executable:
+          executables.add(file)
+        
+        if file notin pkg.updated:
```

Comment:
This section will need to be indented so only executable files are recompiled.

Reply:
Me and my indents.  Fixed on the next commit

Reply:
Binaries removed